tools/SourceKit/CMakeLists.txt

```diff
@@ -234,6 +238,11 @@ macro(add_sourcekit_executable name)
       set_target_properties(${name}
         PROPERTIES
         LINK_FLAGS "-Wl,-exported_symbol,_main")
+
+      if(SWIFT_ANALYZE_CODE_COVERAGE)
+        set_property(TARGET "${name}" APPEND_STRING PROPERTY
```

Comment:
Oh, those calls to `set_target_properties` look wrong to me.  They should append instead of replacing.